mu-plugins/central-hub/tests/phpunit/unit/meta-data/registerMetaBoxes.php

```diff
@@ -35,9 +35,9 @@ protected function setUp() {
 	/*
 	 * Test register_meta_boxes() should register to add_action( 'admin_menu' ) when event fires.
 	 */
-	public function test_function_should_register_to_action_hook_when_event_fires() {
-		$this->assertTrue( has_action( 'admin_menu', 'spiralWebDB\Metadata\register_meta_boxes' ) );
-	}
+//	public function test_function_should_register_to_action_hook_when_event_fires() {
```

Comment:
@rgadon107 Please remove this test as it's not needed.

Reply:
Done.